remotecv/utils.py

```diff
@@ -10,10 +10,60 @@
 
 import logging
 
+from redis import Redis, Sentinel
+
 
 class Config:
     pass
 
 
 logger = logging.getLogger("remotecv")  # pylint: disable=invalid-name
 config = Config()  # pylint: disable=invalid-name
+
+
+def redis_client():
+    if config.redis_mode == "single_node":
+        return __redis_single_node_client()
+
+    if config.redis_mode == "sentinel":
+        return __redis_sentinel_client()
+
+    return None
+
+
+def __redis_sentinel_client():
+    instances_split = config.redis_sentinel_instances.split(",")
+    instances = [tuple(instance.split(":")) for instance in instances_split]
```

Comment:
Nit: `Sentinel` can be instantiated with instances being a list of lists, instead of a list of tuples. That means one less funciton call, one less cognitive element degrading readability.